main.go

```diff
@@ -192,6 +193,11 @@ func runAudit(c conf.Configuration, auditPath string, outputFile string, outputU
 		panic(err)
 	}
 
+	if setExitCode && auditData.ClusterSummary.Results.Totals.Errors > 0 {
```

Comment:
Sorry, I should have caught this before - this block should go at the very bottom of the function. That way we'll still get to see the audit output.

Reply:
@bobby-brennan I moved that block to the bottom of the function. 😄